components/help/components/formatting.tsx

```diff
@@ -235,7 +235,10 @@ export default function HelpFormatting(): JSX.Element {
             </p>
             <FormattedMessage
                 id='help.formatting.linkEx'
-                defaultMessage={'[Check out Mattermost!](https://mattermost.com/)'}
+                defaultMessage={'<link>Check out Mattermost!</link>'}
+                values={{
+                    link: (msg: React.ReactNode) => (<a href='https://mattermost.com/' referrer='noreferrer' target='_blank'>{msg}</a>),
```

Comment:
I think we actually want too revert this one. It's just using a regular `FormattedMessage`, and we want the raw Markdown text here since it's an example of how to use Markdown. This whole piece of code does look rather weird as well, but that's not something to fix right now.